packages/provwasm-std-derive/README.md

```diff
@@ -0,0 +1,5 @@
+# osmosis-std-derive
```

Comment:
Very minor, but do we need to rename this stuff?

Reply:
yep, it's been removed in favor of a rewrite